fdbserver/RestoreApplier.actor.cpp

```diff
@@ -96,11 +96,11 @@ ACTOR static Future<Void> handleSendMutationVectorRequest(RestoreSendVersionedMu
                                                           Reference<RestoreApplierData> self) {
 	// Assume: self->processedFileState[req.fileIndex] will not be erased while the actor is active.
```

Comment:
```suggestion
	// Assume: self->processedFileState[req.asset] will not be erased while the actor is active.
```
It seems `req.fileIndex` has been replaced with `req.asset`